iroh/src/node.rs

```diff
@@ -465,7 +465,6 @@ impl<D: iroh_blobs::store::Store> NodeInner<D> {
             // connections: Operations will immediately fail with ConnectionError::LocallyClosed.
             // All streams are interrupted, this is not graceful.
             self.endpoint
-                .clone()
```

Comment:
lol, frankly this is probably sufficient reason to not make this move the endpoint.